### PR TITLE
HARP-6605: Added map-repeat param to MapView.

### PR DESCRIPTION
**Pull request by bohdanhorpynchuk**
Signed-off-by: bohdanhorpynchuk <ext-bogdan.gorpynchuk@here.com>

### CODE REVIEW

File: @here/harp-mapview/lib/MapView.ts [hunk 1]
```diff
@@ -506,6 +506,13 @@ export interface MapViewOptions {
      */
     enablePhasedLoading?: boolean;
 
+    /**
+     * Enable map repeat. If `false`, map will be repeated on X axis continuously.
+     *
+     * @default `true`
+     */
+    mapRepeat?: boolean;
```
Comment (ninok) [inline @here/harp-mapview/lib/MapView.ts]: mapRepeat is not very descriptive name. Maybe stick to tileWrappingEnabled

File: @here/harp-mapview/lib/MapView.ts [hunk 2]
```diff
@@ -506,6 +506,13 @@ export interface MapViewOptions {
      */
     enablePhasedLoading?: boolean;
 
+    /**
+     * Enable map repeat. If `false`, map will be repeated on X axis continuously.
```
Comment (ninok) [inline @here/harp-mapview/lib/MapView.ts]: 1. If true, the map will repeat
2. only for planar projections
3. instead of X axis I would reword it in longitudal direction (i.e. there is no X axis in geo space)
4. also mention where it will repeat/wrap around (i.e. data-border or antimeridian)

File: @here/harp-mapview/test/VisibleTileSetTest.ts [hunk 3]
```diff
@@ -44,7 +44,8 @@ class Fixture {
         this.frustumIntersection = new FrustumIntersection(
             this.camera,
             MapViewDefaults.projection,
-            MapViewDefaults.extendedFrustumCulling
+            MapViewDefaults.extendedFrustumCulling,
+            true
```
Comment (nzjony) [inline @here/harp-mapview/test/VisibleTileSetTest.ts]: add a test for false please
